packages/gatsby-theme/src/components/Skip/SkipLink.js

```diff
@@ -0,0 +1,44 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { Box } from "theme-ui"
+
+const SkipLink = ({ children, id, ...props }) => {
+  return (
+    <Box
+      as='a'
+      tabindex='0'
+      href={`#${id}`}
+      __css={{
+        display: "flex",
+        alignItems: "center",
+        bg: "#F9F9F9",
+        height: 8,
+        top: "0",
+        left: "10%",
+        zIndex: 100,
+        padding: "8px",
+        position: "absolute",
```

Comment:
We should use the `theme-ui` values declared for the padding. We should also use the shorthands of properties by our conventions (`p` instead of `padding`)